app/controllers/state_file/questions/nj_review_controller.rb

```diff
@@ -1,6 +1,44 @@
 module StateFile
   module Questions
     class NjReviewController < BaseReviewController
+      before_action :get_detailed_return_info
+
+      def get_detailed_return_info
```

Comment:
[DUST] Is there a way to pull the XML values rather than the calculated values? This is something that CfA made a point of during our pairing sessions. Since XML is what we actually submit, we'd ideally use those rather than calculator values.

Reply:
Not sure who you paired with John, and I may not be aware of a planned refactor, but the other "detail" sections for (at least some of) the other states pull from the calculator as well.

Reply:
Also, I appreciate this refactor from explicitly referencing each value in the view.

Reply:
The stuff I paired on was specifically the PDF generation, so good to know that the review screen pulls from the calculator!